script/seed.js

```diff
@@ -1,98 +1,264 @@
     }),
     Product.create({
-      name: 'Money Tree',
+      name: "Money Tree",
       imageUrl:
-        'https://cdn.apartmenttherapy.info/image/upload/f_auto,q_auto:eco,c_fit,w_730,h_486/project%20prism%2Fcolor%20search%20archive%2F95356a0830788aa37fa166d9d8818fbada3ebb89',
+        "https://cdn.apartmenttherapy.info/image/upload/f_auto,q_auto:eco,c_fit,w_730,h_486/project%20prism%2Fcolor%20search%20archive%2F95356a0830788aa37fa166d9d8818fbada3ebb89",
       price: 65.77,
       desc: `If you’ve ever noticed a little potted tree with an unusual braided trunk (that’s one on the far left in the photo above), you’ve encountered a money tree. The trunk braid and leaves have symbolism for many people who believe that they bring good luck and financial success.`,
     }),
     Product.create({
-      name: 'Rubber Plant',
+      name: "Rubber Plant",
       imageUrl:
-        'https://cdn.apartmenttherapy.info/image/upload/f_auto,q_auto:eco,c_fit,w_730,h_487/project%20prism%2Fcolor%20search%20archive%2F89f5a8844ff040228052e92b31ca867051e36b01',
+        "https://cdn.apartmenttherapy.info/image/upload/f_auto,q_auto:eco,c_fit,w_730,h_487/project%20prism%2Fcolor%20search%20archive%2F89f5a8844ff040228052e92b31ca867051e36b01",
       price: 88.99,
       desc: `In recent years, the ubiquitous fiddle leaf fig has dominated all other entrants in the competition for most popular in-house tree-like plants. But we think it’s time to give this understated but stately tree — with its large, deep green leaves — a second look and another chance. Here’s how to maintain a rubber plant as a striking part of your home decor for years to come.`,
     }),
     Product.create({
-      name: 'String of Pears',
+      name: "String of Pears",
       imageUrl:
-        'https://cdn.apartmenttherapy.info/image/upload/f_auto,q_auto:eco,c_fit,w_730,h_487/project%20prism%2Fcolor%20search%20archive%2F5e06e150da339f84d1b4b795655e524f485c7679',
+        "https://cdn.apartmenttherapy.info/image/upload/f_auto,q_auto:eco,c_fit,w_730,h_487/project%20prism%2Fcolor%20search%20archive%2F5e06e150da339f84d1b4b795655e524f485c7679",
       price: 22.56,
       desc: `Not to throw shade at other succulents, but the string of pearls (Senecio rowleyanus) succulent is a show-stopper. With its overflowing vines and bubbly pearl-like leaves, it’s an eye catcher that looks beautiful trailing across a desk or bookshelf or even hanging in a sunny window, where the vines can trail down for interest. Although these plants look delicate, they’re part of the succulent family—a generally easy-to-care-for group of plants. This one in particular is a sure winner for a special space by your window. Read on for tips for growing and caring for this special plant.`,
     }),
   ]);
 
+  // USER PRODUCTS (PRODUCTS PURCHASED & PRODUCTS IN CART ORGANIZED BY USER)
+  const userProducts = await Promise.all([
+    userProducts.create({
```

Comment:
line 99 and all other creates for userProducts needs to be capitalized. Like this UserProducts (or the name of your model)